.github/workflows/clear-cache.yml

```diff
@@ -0,0 +1,22 @@
+name: Weekly cache-clear
+
+on:
+  schedule:
+    - cron: "0 1 * * SUN" # At 01:00 on Sunday
+  workflow_dispatch:
+  
+jobs:
+  delete-cache:
+    runs-on: [self-hosted, linux]
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Delete cached iso files
+        run: rm -rf /usr/share/runner-dependencies/packer_cache/*
+        
+      - name: Delete any remaining VM artifacts
+        run: rm -rf /usr/share/runner-dependencies/virtual_machines/*
+        
+      - name: Delete any remaining venv artifacts
+        run: rm -rf /usr/share/runner-dependencies/socbed_env || true
```

Comment:
Why do we need this `|| true` here?

Reply:
`socbed_env` "should" not exist anymore, since it should be deleted at the end of every system test.
This is just a failsafe in case something went wrong, and the `|| true` forces this specific command to always return `0`m otherwise the action would show up as failed.